tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py

```diff
@@ -56,16 +67,36 @@ def run(ceph_cluster, **kw):
     if w.expired:
         raise RemoveOsdError("Failed to perform osd rm operation. Timed out!")
 
-    # Get osd tree and check whether the deleted osd is present or not
-    out = CephAdm(osd_node).ceph.osd(command="tree", format="json")
-    osd_tree = loads(out[0])
-    for osd_ in osd_tree.get("nodes"):
-        if osd_.get("id") == osd_id and osd_.get("status") == "up":
-            raise RemoveOsdError("The osd is up after rm operation")
+    # Step to validate lvm zap
+    if config.get("operation"):
+        timeout, interval = 60, 5
+        for w in WaitUntil(timeout=timeout, interval=interval):
+            try:
+                cmd = f" ceph-volume lvm zap --destroy {osd_lvm}"
```

Comment:
Can we have cli library for this command ?

Reply:
This command has  to be executed only using  podman and not via cephadm shell. In that case its better to be kept outside of cli? What do you think Vaibhav?

Reply:
missed it